src/mqtt.cpp

```diff
@@ -107,6 +108,43 @@ void mqttmsg_callback(MongooseString topic, MongooseString payload) {
       divertmode_update(newdivert);
     }
   }
+  else if (topic_string == mqtt_topic + "/charge_current/set")
+  {
+    int newchargecurrent = payload_str.toInt();
+    DBUGF("Set charge_current: %d", newmode);
+    //using override, the only interest of this is having priority above everything else.
+    //We have a confusion in web interface to solve as details of the override are not displayed
+    EvseProperties props(EvseState::Active);
+    props.setChargeCurrent(newchargecurrent);
+    evse.claim(EvseClient_OpenEVSE_Manual, EvseManager_Priority_Manual, props);
```

Comment:
I think the priority should be the same as the HTTP API and there should probably be a specific client for these changes, eg

```suggestion
    evse.claim(EvseClient_OpenEVSE_MQTT, EvseManager_Priority_API, props);
```

Reply:
I was using a dedicated MQTT id first, then removed it, to match what http interface did :)
It seems http://openevse/override use EvseClient_OpenEVSE_Manual, EvseManager_Priority_Manual ( I've copied it )

Reply:
I've pushed it with new EvseClient_OpenEVSE_MQTT, EvseManager_Priority_MQTT (500)

Reply:
I've tested, but it has collaterals.
Now, as priority has changed, it is not considered as an override, but just a claim.
So on web interface we can't see and delete it.
Also manual_override is not set to 1 then on mqtt. 

Should we expose claims over mqtt ? 
Do you have some suggestions ?

Reply:
I've opened an issue for this to not spam too much this page : https://github.com/OpenEVSE/ESP32_WiFi_V4.x/issues/373